templates/scenarios/ts/spfx-tab-pre/.vscode/tasks.json

```diff
@@ -31,52 +31,46 @@
             }
         },
         {
-            // Build and upload Teams manifest.
-            // See https://aka.ms/teamsfx-debug-prepare-manifest-task to know the details and how to customize the args.
-            "label": "Build & upload Teams manifest",
+            // Create the debug resources.
+            // See https://aka.ms/teamsfx-provision-task to know the details and how to customize the args.
+            "label": "Create resources",
             "type": "teamsfx",
-            "command": "debug-prepare-manifest",
+            "command": "provision",
             "args": {
-                //// Enter your own Teams app package path if using the existing Teams manifest. ////
-                // "appPackagePath": ""
+                "template": "${workspaceFolder}/teamsfx/app.local.yml",
+                "env": "local"
             }
         },
         {
-            // Check if all the npm packages are installed and will install them if not.
-            // See https://aka.ms/teamsfx-npm-package-task to know the details and how to customize the args.
-            "label": "Install npm packages",
+            // Set up local projects.
+            // See https://aka.ms/teamsfx-deploy-task to know the details and how to customize the args.
+            "label": "Set up local projects",
             "type": "teamsfx",
-            "command": "debug-npm-install",
+            "command": "deploy",
             "args": {
-                "projects": [
-                    {
-                        "cwd": "${workspaceFolder}/SPFx",
-                        "npmInstallArgs": [
-                            "--no-audit"
-                        ]
-                    }
-                ]
+                "template": "${workspaceFolder}/teamsfx/app.local.yml",
+                "env": "local"
             }
         },
         {
             "label": "gulp trust-dev-cert",
             "type": "process",
             "command": "node",
             "args": [
-                "${workspaceFolder}/SPFx/node_modules/gulp/bin/gulp.js",
```

Comment:
SPFx solution is located at `${workspaceFolder}/src`, should be `${workspaceFolder}/src/node_modules/gulp/bin/gulp.js`